gobblin-core/src/main/java/org/apache/gobblin/source/extractor/watermark/DateWatermark.java

```diff
@@ -108,9 +116,9 @@ public int getDeltaNumForNextWatermark() {
   /**
    * recalculate interval(in hours) if total number of partitions greater than maximum number of allowed partitions
    *
-   * @param difference in range
-   * @param hour interval (ex: 4 hours)
-   * @param Maximum number of allowed partitions
+   * @param diffInMilliSecs in range
```

Comment:
Should this be `diffInMilliSecs difference in range`?
Same for the other parameters and in the files.